R/write_validation_report.R

```diff
@@ -0,0 +1,94 @@
+#' Create validation report from template
+#' @param pkg_name name of package
+#' @param template what validation report template from {valtools} to use,
+#' passed to \code{usethis::use_template}
+#' @param open boolean whether to open the validation report, passed to \code{usetthis::use_template}
+#' @importFrom usethis use_template proj_set
+#' @importFrom tools file_ext
+#' @importFrom rlang with_interactive
+#' @export
+vt_use_report <- function(pkg_name = desc::desc_get_field("Package"),
+                          template = "validation_report.Rmd",
```

Comment:
atm, yes but we are planning a future one to capture the use case where someone is circulating just requirements or requirements+test cases for signatures.

Reply:
I think it might be useful to abstract it a but and not give them direct access to defining the template, just select "validation_report","requirement_report","test_plan_report" - where we have control over those templates rather than the user having to know what it is called in our inst dir.